doc/user/content/get-started.md

```diff
@@ -138,172 +132,307 @@ The first step is to declare where to find and how to connect to the sample Kafk
-        SUM(purchase_price) AS revenue,
-        COUNT(id) AS orders,
-        MAX(created_at::timestamp) AS latest_order
-      FROM purchases
-      GROUP BY item_id;
+    SELECT * FROM mz_internal.mz_source_statistics;
     ```
+    <p></p>
 
-    As in other databases, a view doesn't store the results of the query, but simply provides an alias for the embedded `SELECT` statement.
+    ```nofmt
+     id  | worker_id | snapshot_committed | messages_received | updates_staged | updates_committed | bytes_received
+    -----+-----------+--------------------+-------------------+----------------+-------------------+----------------
+     u8  | 0         | t                  | 6236924           | 3221400        | 3221400           | 727364017
+     u10 | 0         | t                  | 2000              | 2000           | 2000              | 401564
+    (2 rows)
+    ```
 
-    To keep the results of this view incrementally maintained **in memory** within our cluster, we'll create an [index](/sql/create-index).
+    Once `snapshot_committed` is `t` for both sources, move on to the next step. -->
 
-1. Create an index `item_purchases_idx`:
+## Step 4. Compute real-time results
 
-   ```sql
-   CREATE INDEX item_purchases_idx ON qck.item_purchases (item_id);
-   ```
+With source data streaming in, you can now explore the unique value of Materialize: computing real-time results over fast-changing data.
 
-1. To see the results:
+1. Switch to your `compute` cluster:
 
     ```sql
-    SELECT * FROM item_purchases LIMIT 10;
+    SET CLUSTER = compute;
+    ```
+
+1. Enable `psql`'s timing feature so you can see how quickly results are returned:
 
-    --Lookups will be fast because of the index on item_id!
-    SELECT * FROM item_purchases WHERE item_id = 768;
+    ```sql
+    \timing
     ```
 
-    Regardless of how complex the underlying view definition is, querying an **indexed** view is computationally free because the results are pre-computed and available in memory. Using indexes can help decrease query time, as well as optimize other operations that depend on the view (like joins).
+1. Materialize is receiving a continuous stream of `purchase` updates. Let's create a [view](/overview/key-concepts/#non-materialized-views) joining data from `purchases` and `items` to count the purchases in each item category:
+
+    ```sql
+    CREATE VIEW purchases_per_category AS
+      SELECT category, count(*)
+      FROM items
+      JOIN purchases ON items.id = purchases.item_id
+      GROUP BY items.category;
+    ```
 
-### Enrich data with joins
+    As in other SQL databases, a view doesn't store the results of the query but simply provides an alias for the embedded `SELECT` statement.
 
-Materialize efficiently supports [all types of SQL joins](/sql/join/#examples) under all the conditions you would expect from a relational database, including maintaining strong [consistency guarantees](/overview/isolation-level/).
+1. Run the view a few times:
```

Comment:
This seems more confusing to me because we're not running the embedded select in the view but rather running the view. I prefer to leave this as-is for now.